exercises/practice/diffie-hellman/diffie_hellman_test.go

```diff
@@ -1,3 +1,4 @@
+ //nolint:gosec // Randomness is part of an exercise where math.rand is more appropriate than crypto.rand
```

Comment:
Why does the test file need this? I don't see an import for math.rand.

Reply:
this was a mistake, I fixed it